src/components/views/sul-study-place/filtering-table/study-place-filtering-table.tsx

```diff
@@ -240,7 +240,7 @@ const StudyPlaceFilteringTable = ({items}: Props) => {
                   )}
                 </Td>
                 <Td className="block w-auto sm:border-b sm:border-black-40 md:text-left lg:table-cell lg:w-1/5">
-                  {item.libCalId && (
+                  {item.libCalId ? (
```

Comment:
my personal preference is to use two separate conditions:
```
{foo && <div>foo</div>}
{!foo && <div>bar</div>}
```

What do you think?

Reply:
Sounds good to me! Separating out the conditionals makes it easier to read 😎